CI_YAML.md

```diff
@@ -160,44 +283,210 @@ dependencies: >-
     {"dependency": "goldctl"}
   ]
 ```
+</td>
+  </tr>
+  <tr>
+    <td>device_type</td>
+    <td>The phone device type a target will be running against. For host only targets that do not need
+    a phone, a `none` value should be used.
```

Comment:
consider: 'a value of 'none' should be used.

Reply:
Done.